src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -84,4 +76,24 @@ protected void runTool(final JavaSparkContext ctx) {
         assembledBreakpoints.saveAsTextFile(output);
     }
 
+    static JavaPairRDD<String, ContigsCollection> getContigsCollectionKeyedByBreakpointId(final JavaSparkContext ctx, final String inputPath) {
```

Comment:
What about a documentation
"Load a single local assembly flat file and turn into {@link ContigsCollection}, keyed by assembly id."?